client/src/components/Profile/MentorStatsCard.tsx

```diff
@@ -100,8 +105,7 @@ export function MentorStatsCard(props: Props) {
                                     marginBottom: 5,
                                   }}
                                 >
-                                  <a href={`/profile?githubId=${githubId}`}>{name}</a>{' '}
-                                  {isExpelled ? <Tag color="red">expelled</Tag> : <Tag color="green">active</Tag>}
```

Comment:
Hiding `expel` tag will not hide the item itself, if it is exists in the student's array. Removing this data should be done only after updating the backend part.

In this implementation expelled by mas expel process student will be shown in the list with the status 'Active'.